setpass/api.py

```diff
@@ -166,5 +168,46 @@ def add(user_id):
     return Response(response=user.token, status=200)
 
 
+@wsgi.app.route('/reset', methods=['GET'])
+def view_reset_form():
+    return render_template('reset_form.html')
+
```

Comment:
theoretically these two views can be combined, i.e. `methods=['GET', POST']` then inside the function do `if request.method == 'POST':` etc (`from flask import request` needed)  

again, not sure if it's really cleaner one way or the other, just thought I would mention